docker/pravega/scripts/init_kubernetes.sh

```diff
@@ -47,6 +47,13 @@ init_kubernetes() {
         local podname=${POD_NAME}
         export PUBLISHED_ADDRESS=""
         export PUBLISHED_PORT=""
+        local service=$( k8 "${ns}" "services" "${podname}" .kind )
+
+        while [[  "$service" != "Service" ]]; 
```

Comment:
as discussed with Ravi, I see this as a Precondition check before starting Segment store. And the startup  on k8s should not happen if the precondition fails.

Reply:
I have removed the loop and have added exit condition now.